webapp/channels/src/components/admin_console/team_channel_settings/channel/details/channel_moderation.tsx

```diff
@@ -21,172 +21,150 @@ import {ChannelModerationRoles} from './types';
             id: t('admin.channel_settings.channel_moderation.channelMentions.disabledMember'),
             defaultMessage: 'Channel mentions for members are disabled in [{scheme_name}](../permissions/{scheme_link}).',
             permissionName: 'use_channel_mentions',
-        },
-        disabledBoth: {
+        }),
+        disabledBoth: defineMessage({
             id: t('admin.channel_settings.channel_moderation.channelMentions.disabledBoth'),
             defaultMessage: 'Channel mentions for members and guests are disabled in [{scheme_name}](../permissions/{scheme_link}).',
             permissionName: 'use_channel_mentions',
-        },
-        disabledGuestsDueToCreatePosts: {
+        }),
+        disabledGuestsDueToCreatePosts: defineMessage({
             id: t('admin.channel_settings.channel_moderation.channelMentions.disabledGuestsDueToCreatePosts'),
             defaultMessage: 'Guests can not use channel mentions without the ability to create posts.',
-        },
-        disabledMembersDueToCreatePosts: {
+        }),
+        disabledMembersDueToCreatePosts: defineMessage({
             id: t('admin.channel_settings.channel_moderation.channelMentions.disabledMemberDueToCreatePosts'),
             defaultMessage: 'Members can not use channel mentions without the ability to create posts.',
-        },
-        disabledBothDueToCreatePosts: {
+        }),
+        disabledBothDueToCreatePosts: defineMessage({
             id: t('admin.channel_settings.channel_moderation.channelMentions.disabledBothDueToCreatePosts'),
             defaultMessage: 'Guests and members can not use channel mentions without the ability to create posts.',
-        },
-    },
-
-    title: {
-        id: t('admin.channel_settings.channel_moderation.title'),
-        defaultMessage: 'Channel Moderation',
-    },
-    subtitle: {
-        id: t('admin.channel_settings.channel_moderation.subtitle'),
-        defaultMessage: 'Manage the actions available to channel members and guests.',
-    },
-    subtitleMembers: {
-        id: t('admin.channel_settings.channel_moderation.subtitleMembers'),
-        defaultMessage: 'Manage the actions available to channel members.',
-    },
-    permissions: {
-        id: t('admin.channel_settings.channel_moderation.permissions'),
-        defaultMessage: 'Permissions',
-    },
-    guests: {
-        id: t('admin.channel_settings.channel_moderation.guests'),
-        defaultMessage: 'Guests',
-    },
-    members: {
-        id: t('admin.channel_settings.channel_moderation.members'),
-        defaultMessage: 'Members',
+        }),
     },
 });
 
-interface Props {
-    channelPermissions?: ChannelPermissions[];
-    onChannelPermissionsChanged: (name: string, channelRole: ChannelModerationRoles) => void;
-    teamSchemeID?: string;
-    teamSchemeDisplayName?: string;
-    guestAccountsEnabled: boolean;
-    isPublic: boolean;
-    readOnly?: boolean;
-}
+const titleMessage = defineMessage({
```

Comment:
I would put all these in one single `defineMessages`.

Reply:
pulled into single defineMessages